packages/@aws-cdk/aws-pipes-targets-alpha/test/integ.api-gateway.ts

```diff
@@ -0,0 +1,75 @@
+import { IPipe, ISource, Pipe, SourceConfig } from '@aws-cdk/aws-pipes-alpha';
+import { ExpectedResult, IntegTest } from '@aws-cdk/integ-tests-alpha';
+import * as cdk from 'aws-cdk-lib';
+import * as apigw from 'aws-cdk-lib/aws-apigateway';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import { ApiGatewayTarget } from '../lib/api-gateway';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'aws-cdk-pipes-targets-api-gw');
+const sourceQueue = new cdk.aws_sqs.Queue(stack, 'SourceQueue');
+
+// SQS (pipe source) --> API Gateway REST API (pipe target) --> Lambda function
+
+class TestSource implements ISource {
+  sourceArn: string;
+  sourceParameters = undefined;
+  constructor(private readonly queue: cdk.aws_sqs.Queue) {
+    this.queue = queue;
+    this.sourceArn = queue.queueArn;
+  }
+  bind(_pipe: IPipe): SourceConfig {
+    return {
+      sourceParameters: this.sourceParameters,
+    };
+  }
+  grantRead(pipeRole: cdk.aws_iam.IRole): void {
+    this.queue.grantConsumeMessages(pipeRole);
+  }
+}
+
+const fn = new lambda.Function(stack, 'ConnectHandler', {
+  runtime: lambda.Runtime.NODEJS_LATEST,
+  handler: 'index.handler',
+  code: new lambda.InlineCode('exports.handler = async function(event, context) { console.log(event); return { statusCode: 200, body: "connected" }; };'),
+});
+
+const restApi = new apigw.RestApi(stack, 'RestApi', {});
+restApi.root.addResource('books').addResource('fiction').addMethod('GET', new apigw.LambdaIntegration(fn));
+
+new Pipe(stack, 'Pipe', {
+  source: new TestSource(sourceQueue),
+  target: new ApiGatewayTarget(restApi, {
+    method: 'GET',
+    path: '/books/*',
+    headerParameters: {
+      'x-header': 'myheader',
+    },
+    queryStringParameters: { key: 'USA' },
+    pathParameterValues: ['fiction'],
```

Comment:
@nmussy I think this is what you're after? `fiction` replaces the `*` in `path` defined above. More coverage..